### ISSUE TITLE
All Pages - Missing Language Type

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#11]
While testing using the DHS Trusted Tester process, I found all the pages are missing a language type which should be **lang="en"** and part the HTML tag.

See example below:

<html lang="en">

Section 508 - 31(a) and 31(b)
WCAG 2.0 - 9.1 Web: Language

**Comment by adamsfallen** [comment]
Added - c24d99f355909cd62bbfda6496e5b618e6537107
